universal-application-tool-0.0.1/project/TailwindBuilder.scala

```diff
@@ -12,6 +12,14 @@ object TailwindBuilder {
         process = Option(
           Process("npx tailwindcss build -i ./app/assets/stylesheets/styles.css -o ./public/stylesheets/tailwind.css", base).run()
         )
+
+        process = Option(
+          Process("npx grunt dist", base).run()
+        )
+
+        process = Option(
```

Comment:
Yeah it might be better to use just plain sbt over grunt. It seems more elegant than adding another layer of code watching. I tried that at first but it was pretty confusing, but I'll take a look at these links you sent me and I will reach out for more help if I can't figure it out

Reply:
Thank you I appreciate this